src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
Thanks. Thinking of adding something like this:

<!--
        Served over HTTP:
			feeds.geekzone.co.nz
			m.geekzone.co.nz
			price.geekzone.co.nz

		Served over HTTP, some pages on HTTPS (registration, login, private messages)
			www.geekzone.co.nz

		Available over HTTPS only (301 redirect from HTTP)
			live.geekzone.co.nz
			testip.geekzone.co.nz

		Available over HTTP and HTTPS
			cdn.geekzone.co.nz
			images.geekzone.co.nz
-->

Is it clear enough? 

Do we need to list specific pages that are available over HTTPS on the main domain, even though it is mainly served over HTTP?